binary-manager.js

```diff
@@ -0,0 +1,169 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+const { Cu } = require("chrome");
+const { OS } = require("resource://gre/modules/osfile.jsm");
+const { Services } = Cu.import("resource://gre/modules/Services.jsm", {});
```

Comment:
Right! Unfortunately, we can't do that here yet, since this add-on might be loaded be older versions of Firefox, at least back to release (maybe ESR too...?). So, we'd need to wait for `ChromeUtils` to exist in all those places first. I filed a [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1440692) for this.